### ISSUE TITLE
User identity can not be set

### ISSUE THREAD

**Issue by SleeplessByte** [BranchMetrics/Branch-Android-SDK#94]
````
I/BranchMetricsLoggedInBehavior﹕ [branch-behavior] branch identity result 
{
    "error_message": "Trouble reaching server. Please try again in a few minutes"
}

Trouble setting the user alias. Check network connectivity and that you properly initialized
````

Oh, and I am connected. For sure.

**Comment by aaustin** [comment]
This must be related to your initialization woes.. This works for me on the TestBed app - will continue convo in other issue

**Comment by sojanpr** [comment]
@SleeplessByte  thanks for posting out the exact issue. Fix is now available with v1.6.0 .
